srcDoc/examples.rst

```diff
@@ -404,24 +404,24 @@ is recommended to reduce the send Queue to lower values.
 By using the parameter ``-pubRate <samples/s>``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-This parameter can be used in order to limit the frequency in which *RTI
-Perftest* publishes samples. This can help reducing the number of
-packages in the network, helping to achieve better Latency and
-Throughput numbers.
+This parameter can be used to limit the frequency at which *RTI
+Perftest* publishes samples. This can help reduce the number of
+packages in the network, achieving better latency and
+throughput numbers.
 
 By using a flow controller ``-flowController <default,1Gbps,10Gbps>``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Since the sample size is bigger than 63000 Bytes, *RTI Perftest* will
-enable Asynchronous Publing. By enabling that, you also make use of the
-default *Flow Controller*, which might not be optimal. It might be a
-good practice to specify a *Flow Controller* tailored for the network
+enable Asynchronous Publishing. By enabling Asynchronous Publishing, you also make use of the
+default *FlowController*, which might not be optimal. It might be a
+good practice to specify a *FlowController* tailored for the network
```

Comment:
optimal. Therefore it is a good practice to also specify a *FlowController* that fits to the characteristics (Bandwidth, latency, etc) of the network where the *RTI Perftest* applications are going to run.

Reply:
done